### PR TITLE
[Rule Tuning] Windows DR Tuning - 8

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

Part of #3186 

## Summary

Tunes the following rules:

* Suspicious .NET Reflection via PowerShell
* Suspicious WMIC XSL Script Execution
* Unusual Child Process from a System Virtual Process
* File Creation Time Changed
* Potential Evasion via Filter Manager

### CODE REVIEW

File: rules/windows/defense_evasion_suspicious_wmi_script.toml [hunk 1]
```diff
@@ -28,7 +28,7 @@ sequence by process.entity_id with maxspan = 2m
 [process where host.os.type == "windows" and event.type == "start" and
    (process.name : "WMIC.exe" or process.pe.original_file_name : "wmic.exe") and
    process.args : ("format*:*", "/format*:*", "*-format*:*") and
-   not process.command_line : "* /format:table *"]
+   not process.command_line : ("* /format:table *", "* /format:table")]
```
Comment (Aegrah) [inline rules/windows/defense_evasion_suspicious_wmi_script.toml:31]: Am I crazy, or should `* /format:table *` match everything that `* /format:table` matches and more? Or does that specific space character do the trick? If so, wouldn't it make more sense to change it to `* /format:table*`?

File: rules/windows/defense_evasion_unusual_system_vp_child_program.toml [hunk 2]
```diff
@@ -31,7 +31,7 @@ type = "eql"
 
 query = '''
 process where host.os.type == "windows" and event.type == "start" and
-  process.parent.pid == 4 and
+  process.parent.pid == 4 and process.executable: "?*" and
```
Comment (Aegrah) [inline rules/windows/defense_evasion_unusual_system_vp_child_program.toml:34]: Just want to check that process.executable is not empty? Wouldn't `process.executable != null` work there?

Comment (Aegrah) [inline rules/windows/defense_evasion_unusual_system_vp_child_program.toml:34]:
```suggestion
  process.parent.pid == 4 and process.executable : "?*" and
```

File: rules/windows/defense_evasion_via_filter_manager.toml [hunk 3]
```diff
@@ -103,7 +103,19 @@ type = "eql"
 
 query = '''
 process where host.os.type == "windows" and event.type == "start" and
- process.name : "fltMC.exe" and process.args : "unload"
+  process.name : "fltMC.exe" and process.args : "unload" and
+  not
+  (
+    (
+      process.executable: "?:\\Program Files (x86)\\ManageEngine\\UEMS_Agent\\bin\\DCFAService64.exe" and
```
Comment (Aegrah) [inline rules/windows/defense_evasion_via_filter_manager.toml:110]:
```suggestion
      process.executable : "?:\\Program Files (x86)\\ManageEngine\\UEMS_Agent\\bin\\DCFAService64.exe" and
```

File: rules/windows/defense_evasion_via_filter_manager.toml [hunk 4]
```diff
@@ -103,7 +103,19 @@ type = "eql"
...
+      process.args : ("DFMFilter", "DRMFilter")
+    ) or
+    (
+      process.executable: "?:\\Windows\\SysWOW64\\msiexec.exe" and
```
Comment (Aegrah) [inline rules/windows/defense_evasion_via_filter_manager.toml:114]:
```suggestion
      process.executable : "?:\\Windows\\SysWOW64\\msiexec.exe" and
```

**Review by terrancedejesus — APPROVED** [review]
Ruben had some good comments and suggestions.
Approving.

File: rules/windows/defense_evasion_suspicious_wmi_script.toml (hunk 1 above)
Comment (w0rk3r) [reply rules/windows/defense_evasion_suspicious_wmi_script.toml:31]: It doesn't exclude a command line that ends with `/format:table`, but I'm sure if adding that wildcard could enable someone to abuse it by naming the file as table.xsl

File: rules/windows/defense_evasion_unusual_system_vp_child_program.toml (hunk 2 above)
Comment (w0rk3r) [reply rules/windows/defense_evasion_unusual_system_vp_child_program.toml:34]: > The field != null syntax expects that the field doesn't exist/is not populated in the event

Ref: https://github.com/elastic/sdh-security-team/issues/462#issuecomment-1274562038

195080713-6ef33182-cc22-4f39-bca6-b63639d0009f

**Review by Aegrah — APPROVED** [review]
LGTM!

**Pull request merged**